File: app/src/main/java/com/neome/api/meta/base/dto/DefnLayoutForm.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Symbol;
import com.neome.api.meta.base.Types.EnumDefnFormLayoutType;
import com.neome.api.meta.base.Types.MetaIdLayoutForm;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnLayoutForm
{
  public MetaIdLayoutForm metaId;

  public Symbol name;

  @Nullable
  public EnumDefnFormLayoutType type;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnLayoutForm.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.EnumDefnFormLayoutType
import com.neome.api.meta.base.Types.MetaIdLayoutForm
import com.neome.api.meta.base.Symbol

interface DefnLayoutForm
{
  val metaId: MetaIdLayoutForm
  val name: Symbol
  val type: EnumDefnFormLayoutType?
}